Pokemon/Pokemon/PKPokedexTableViewController.m

```diff
@@ -0,0 +1,481 @@
+//
+//  PKPokedexTableViewController.m
+//  Pokemon
+//
+//  Created by Jamaal on 7/27/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "PKPokedexTableViewController.h"
+#import "PKPokemonClickDetailsViewController.h"
+
+@interface PKPokedexTableViewController ()
+
+@property (nonatomic) NSDictionary *pokemon;
+@property (nonatomic) NSMutableArray *allPokemonAtoZ;
+@property (nonatomic) BOOL showAllPokemon;
+@property (weak, nonatomic) IBOutlet UISegmentedControl *segmentControl;
+@property (nonatomic) NSArray *pokemonIndexAtoZ;
+-(NSArray *)sectionIndexTitlesForTableView:(UITableView *)tableView;
+
+@end
+
+@implementation PKPokedexTableViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    self.pokemonIndexAtoZ = [NSArray arrayWithObjects:@"A", @"B", @"C", @"D", @"E", @"F", @"G", @"H", @"I", @"J", @"K", @"L", @"M", @"N", @"O", @"P", @"Q", @"R", @"S", @"T", @"U", @"V", @"W", @"X", @"Y", @"Z", nil];
+    
+    
+    
+    self.title = @"Ash's Pokedex";
+
+    self.showAllPokemon = NO;
+    
+                    NSArray *bug = @[
```

Comment:
Minor code styling thing - NS Array *bug should be indented the same as everything else on it's level.  For example it should lineup with self.showAllPokemon and NSArray *dragon.

This improves readability.